[2023-09-04T06:02:42.864+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-19T00:00:00+00:00 [queued]>
[2023-09-04T06:02:42.877+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-19T00:00:00+00:00 [queued]>
[2023-09-04T06:02:42.877+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:02:42.893+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-04-19 00:00:00+00:00
[2023-09-04T06:02:42.902+0000] {standard_task_runner.py:57} INFO - Started process 5457 to run task
[2023-09-04T06:02:42.906+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-04-19T00:00:00+00:00', '--job-id', '492', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpf4apgnrj']
[2023-09-04T06:02:42.909+0000] {standard_task_runner.py:85} INFO - Job 492: Subtask insert_into_table
[2023-09-04T06:02:42.979+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-19T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:02:43.086+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-04-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-04-19T00:00:00+00:00'
[2023-09-04T06:02:43.088+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-04-19', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:02:43.098+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:02:43.107+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:02:43.118+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-04-19', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:02:43.122+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:02:43.137+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20220419T000000, start_date=20230904T060242, end_date=20230904T060243
[2023-09-04T06:02:43.197+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:02:43.217+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-04T06:05:30.551+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-19T00:00:00+00:00 [queued]>
[2023-09-04T06:05:30.570+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-19T00:00:00+00:00 [queued]>
[2023-09-04T06:05:30.571+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:05:30.597+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-04-19 00:00:00+00:00
[2023-09-04T06:05:30.607+0000] {standard_task_runner.py:57} INFO - Started process 6788 to run task
[2023-09-04T06:05:30.612+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-04-19T00:00:00+00:00', '--job-id', '1050', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpagp2gm8b']
[2023-09-04T06:05:30.616+0000] {standard_task_runner.py:85} INFO - Job 1050: Subtask insert_into_table
[2023-09-04T06:05:30.722+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-19T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:05:30.968+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-04-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-04-19T00:00:00+00:00'
[2023-09-04T06:05:30.970+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-04-19', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:05:30.988+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:05:31.008+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:05:31.031+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-04-19', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:05:31.039+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:05:31.068+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20220419T000000, start_date=20230904T060530, end_date=20230904T060531
[2023-09-04T06:05:31.145+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:05:31.234+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
